.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,7 +7,7 @@ assignees: ''
 ---
 
 <!--
-This is a bug report tracker. New features are discussed in the forum: https://support.delta.chat
+This is a bug report tracker. New features are discussed in the forum: https://support.delta.chat/c/features/6
```

Comment:
i think, the gist of this line is to redirect  things not matching to the issue tracker to the forum; this is not only feature requests.

so the line could be sth. as "This is a bug report tracker. For feature requests, support and help o to the forum:"

Reply:
@r10s done